game.py

```diff
@@ -20,8 +20,31 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    print(snowman_word)
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES:
+        print_word_progress_string(snowman_word, correct_letter_guess_statuses)
+        user_input = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+        if user_input in correct_letter_guess_statuses:
+            correct_letter_guess_statuses[user_input] = True
+            print("You guessed a letter that's in the word!")
 
+        else:
+            print(f"The letter {user_input} is not in the word")
+            wrong_guesses_list.append(user_input)
+
+        print_snowman_graphic(len(wrong_guesses_list))
+        print(f"Wrong guesses: {wrong_guesses_list}")
```

Comment:
For a better user experience, how could we alter this print statement so that the `[]` aren't wrapped around the guesses of a player? 
```suggestion
        print(f"Wrong guesses: {wrong_guesses_list}")
        
```